payjoin/src/receive/mod.rs

```diff
@@ -1,271 +1,3 @@
-//! Receive Payjoin
-//!
-//! This module contains types and methods used to receive payjoin via BIP78.
-//! Usage is pretty simple:
-//!
-//! 1. Generate a pj_uri [BIP 21](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) using [`payjoin::Uri`](crate::Uri)::from_str
-//! 2. Listen for a sender's request on the `pj` endpoint
-//! 3. Parse the request using [`UncheckedProposal::from_request()`](crate::receive::UncheckedProposal::from_request())
-//! 4. Validate the proposal using the `check` methods to guide you.
-//! 5. Assuming the proposal is valid, augment it into a payjoin with the available `try_preserving_privacy` and `contribute` methods
-//! 6. Extract the payjoin PSBT and sign it
-//! 7. Respond to the sender's http request with the signed PSBT as payload.
-//!
-//! ## Receive a Payjoin
-//!
-//! The `receive` feature provides all of the check methods, PSBT data manipulation, coin selection, and transport structures to receive payjoin and handle errors in a privacy preserving way.
-//!
-//! Receiving payjoin entails listening to a secure http endpoint for inbound requests. The endpoint is displayed in the `pj` parameter of a [bip 21](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) request URI.
-//!
-//! The [reference implementation annotated below](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-cli) uses `rouille` sync http server and Bitcoin Core RPC.
```

Comment:
We can keep this, have it reflect reality, and just change the last line to just link to payjoin-cli